.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "css.validate": false,
+  "less.validate": false,
+  "scss.validate": false
+}
```

Comment:
Maybe we can also add...

-  "editor.defaultFormatter": "esbenp.prettier-vscode",
- "editor.formatOnSave": true,
- "editor.quickSuggestions": {
    "strings": true // for tailwind. https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss
  },